src/Assignments/ProblemIframe.tsx

```diff
@@ -118,13 +113,21 @@ export const ProblemIframe: React.FC<ProblemIframeProps> = ({
                     showSolutions: previewShowSolutions
                 });
             } else {
-                res = await AxiosRequest.get(`/courses/question/${problem.id}${queryString}`);
+                res = await getQuestion({
+                    id: problem.id,
+                    workbookId,
+                    readonly
+                });
             }
             return res.data.data.rendererData.renderedHTML as string;
 
         } catch (e) {
             setError(e.message);
-            logger.error('Error posting preview', e, e.message);
+            
+            if (!BackendAPIError.isBackendAPIError(e) || (e.status !== 200 && e.status !== 400)) {
```

Comment:
```
e.status !== 200 && e.status !== 400
```
Seems confusing to me but I might be overthinking it, I checked and it works as expected 🤷‍♂️